### ISSUE TITLE
Undefined variable: currentCopy in RollbackTask.php

### ISSUE THREAD

**Issue by ParkMasakazu** [andres-montanez/Magallanes#206]
Hi, I am thankful to always use the magallanes.
Thank you for the useful tool.
Recently, I was updated magallanes from v1.0.1 to v1.0.5.
But the following error After rollback occurred in v1.0.5.
```
$ mage releases rollback --release=-1 to:production
PHP Notice: Undefined variable: currentCopy in ./vendor/andres-montanez/magallanes/Mage/Task/BuiltIn/Releases/RollbackTask.php on line 140
```
This is the happy and help solve the problem.
Please continue the development of the future also a good tool.
Thanks you.

**Comment by ZesInternational** [comment]
in Mage\Task\BuiltIn\Releases\RollbackTask at line 91 add:
```php
$currentCopy = $releasesDirectory . '/' . $releaseId;
```
then it will work again.

**Comment by eps90** [comment]
@ZesInternational Could you make a PR with that?

**Comment by eps90** [comment]
@ParkMasakazu Thanks to @ZesInternational the issue is fixed. Please install Magallanes from `develop` branch (in composer.json you need to pass `dev-develop` as version) and check if everything works fine :)

**Comment by ParkMasakazu** [comment]
Hi, It's very quickly.
I have obtained from dev-develop, worked confirmation even here.
Thanks you.

**Comment by eps90** [comment]
@ParkMasakazu So, can we close the issue then?

**Comment by ParkMasakazu** [comment]
OK, No problem. Thanks!

**Comment by eps90** [comment]
Fixed in #209
